### PR TITLE
Remove key duplication in hash

### PR DESCRIPTION
**Pull request by ba1ash**
The key duplication cause an annoying warning. So, fix it.
Also, ignore `Gemfile.lock`, because this is a gem.
Move all dependencies declaration to the gemspec file.
Add `pry` as a dev dependency, bump `nokogiri`.
Make specs workable again.

### CODE REVIEW

**Comment by ba1ash** [comment]
[Eliminate deprecation warnings](https://trello.com/c/8CcLoAvT/687-eliminate-deprecation-warnings)

**Comment by ba1ash** [comment]
:up:

File: lib/mpi_client/option_translator.rb [hunk 1]
```diff
@@ -11,7 +11,6 @@ class OptionTranslator
       :certificate  => :PublicCertificate,
       :private_key  => :PrivateKey,
       :directory_server_url => :DirectoryServerURL,
-      :country_code => :CountryCode,
```
Comment (avsh) [inline lib/mpi_client/option_translator.rb]: why removed?

Comment (ba1ash) [reply lib/mpi_client/option_translator.rb]: we have the same one before in this map. this one cause a warning that we want to eliminate
```
/home/ba1ash/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/bundler/gems/mpi_client-b1922eb72202/lib/mpi_client/option_translator.rb:9: warning: key :country_code is duplicated and overwritten on line
14
```

**Pull request merged**